test/lib/feeder_importer_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
+require 'feeder_importer'
+require 'prx_access'
+
+describe FeederImporter do
+  include PRXAccess
+
+  let(:account_id) { 8 }
+  let(:user_id) { 8 }
+  let(:feeder_podcast_url) { 'https://feeder.prx.org/api/v1/podcasts/40' }
+  let(:importer) { FeederImporter.new(account_id, user_id, feeder_podcast_url) }
+
+  let(:podcast) do
+    api_resource(JSON.parse(json_file('transistor_podcast')), feeder_root).tap do |r|
+      r.headers = r.headers.merge('Authorization' => 'Bearer thisisnotatoken')
+    end
+  end
+
+  before do
+    stub_import_requests
+  end
+
+
+  it "makes a new importer" do
+    importer.wont_be_nil
+  end
+
+  it "retrieves the feeder podcast" do
+    podcast = importer.retrieve_podcast
```

Comment:
maybe name this version of `podcast` something slightly different to set it apart from the podcast you declared on line 13?